storage/src/cache/filecache/mod.rs

```diff
@@ -202,7 +202,7 @@ impl FileCacheEntry {
             is_direct_chunkmap,
             is_stargz
         );
-        let meta = if is_get_blob_object_supported && blob_info.meta_ci_is_valid() {
+        let meta = if blob_info.meta_ci_is_valid() {
```

Comment:
It would be better to tweek value of `is_get_blob_object_supported`, such as
```
let is_get_blob_object_supported = is_v6 && !mgr.is_compressed;
```

Reply:
Oh, seems it would be even better to enforce direct_chunkmap for v6?
It would be better to refine `create_chunk_map()` instead?
```
let (chunk_map, is_direct_chunkmap) =
            Self::create_chunk_map(mgr, &blob_info, &blob_file_path)?;
```

Reply:
But v5 image can also have direct_chunk_map. BTW, how do we define the direct_chunk_map feature?

Reply:
How about this? The `disable_indexed_map` only takes effect for v5 images.
```
diff --git a/storage/src/cache/filecache/mod.rs b/storage/src/cache/filecache/mod.rs
index f92881b2..d387c144 100644
--- a/storage/src/cache/filecache/mod.rs
+++ b/storage/src/cache/filecache/mod.rs
@@ -255,7 +255,8 @@ impl FileCacheEntry {
         // The builder now records the number of chunks in the blob table, so we can
         // use IndexedChunkMap as a chunk map, but for the old Nydus bootstrap, we
         // need downgrade to use DigestedChunkMap as a compatible solution.
-        let chunk_map: Arc<dyn ChunkMap> = if mgr.disable_indexed_map
+        let is_v5 = !blob_info.meta_ci_is_valid();
+        let chunk_map: Arc<dyn ChunkMap> = if (is_v5 && mgr.disable_indexed_map)
             || blob_info.has_feature(BlobFeatures::V5_NO_EXT_BLOB_TABLE)
         {
             direct_chunkmap = false;
```

Reply:
I suppose we don't have to consider what image version (5 or 6) or flag `disable_indexed_map` when loading blob meta info into memory. As long as `blob_info.meta_ci_is_valid()` is True we should load its meta otherwise storage blob io can't work.

Reply:
OK. Make sense.